### PR TITLE
Add September updates

### PR DESCRIPTION
**Pull request by astrojuanlu**
To be sent to NumFOCUS when we merge them.

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]
Everything looking good! 🚀

**Review by Yash-10 — APPROVED** [review]
Yay! Everything looks great :)

**Comment by astrojuanlu** [comment]
Thanks both! Merging :shipit:

File: content/articles/2021-09-september-updates.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Title: August updates
```
Comment (astrojuanlu) [inline content/articles/2021-09-september-updates.md:1]: Wooooooooooooooooooooops 🙈 @jorgepiloto @Yash-10 haha we all missed this

Comment (astrojuanlu) [reply content/articles/2021-09-september-updates.md:1]: Fixed in https://github.com/poliastro/poliastro.github.io/commit/d0ffc71663b603fc6fcba8340f9fba935882a318

**Pull request merged**